sdk/storage/Azure.Storage.Files/src/Models/NtfsAttributes.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Azure.Storage.Files.Models
+{
+    /// <summary>
+    /// NTFS file attributes for Files and Directories.
+    /// </summary>
+    [Flags]
+    public enum NtfsAttributes
+    {
+        /// <summary>
+        /// Clear all flags.
+        /// </summary>
+        None = 0,
+
+        /// <summary>
+        /// The File or Directory is read-only.
+        /// </summary>
+        ReadOnly = FileAttributes.ReadOnly,
```

Comment:
This feels dangerous.  The enum values can't change between versions and it's going to be much riskier if we tie them to another enum.  I'd prefer a switch to map from one to the other.

Reply:
Fixed.